src/components/gitProvider/GitProvider.tsx

```diff
@@ -357,7 +358,7 @@ function GitForm({
     sshPrivateKey = '',
     ...props
 }) {
-    const { state, disable, handleOnChange, handleOnSubmit } = useForm(
```

Comment:
is removal of disable intentional?

Reply:
Yes it was unused and so fixed the code smell